test/tools/ossfuzz/strictasm_diff_ossfuzz.cpp

```diff
@@ -78,7 +78,10 @@ extern "C" int LLVMFuzzerTestOneInput(uint8_t const* _data, size_t _size)
 	ostringstream os2;
 	try
 	{
-		yulFuzzerUtil::interpret(os1, stack.parserResult()->code);
```

Comment:
You don't need to always break everything :-) - if it's short enough to fit into one line, it can stay in one line. Only *if* you want/need to break anything, you then have to break *everything*.